Hardware/index.md

```diff
@@ -4,23 +4,47 @@ SPDX-FileCopyrightText: 2020 seL4 Project a Series of LF Projects, LLC.
 ---
 # Supported Platforms
 
-## Verification status
+### Summary
 
-The seL4 proofs are only for specific platforms, as noted in the *Status* column as follows:
+seL4 is available on 3 major hardware architectures, [ARM](#arm),
+[RISC-V](#risc-v) and [x86](#x86), for a number of platforms (with varying
+hardware features).
+
+seL4 is formally verified (to various degrees) for a subset of these
+configurations, i.e. combination of architecture, platform and hardware
+features.
+
+Details are given in the tables below.
+
+### Platforms' attributes
+
+The tables below list of all the platforms for which seL4's implementation is
+available. For each platform, the tables list:
```

Comment:
```suggestion
The tables below lists the platforms for which seL4 is available. For each platform, the tables list:
```

Reply:
Thanks @lsf37 . I have applied your change except the typo (lists>list)